crates/torii/core/src/processors/event_message.rs

```diff
@@ -47,7 +48,14 @@ where
         // silently ignore if the model is not found
         let model = match db.model(event.keys[MODEL_INDEX]).await {
             Ok(model) => model,
-            Err(_) => return Ok(()),
+            Err(_) => {
+                warn!(
+                    target: LOG_TARGET,
+                    model = %event.keys[MODEL_INDEX],
+                    "System event model not found."
```

Comment:
I feel like it might be confusing referring to event messages as SystemEvents and EventMessages. Should we settle on one unique event name?

Reply:
Ideally as we mentioned during a call, we will actually only have event messages. Which should remove the confusion.
We could switch to `Event message` and keep this nomenclature yeah. Then I'll complete this PR by restricting events to only be available in Dojo and not using Starknet event, wdyt?